nv/rc.py

```diff
@@ -113,3 +115,48 @@ def _parse_line(line: str):
         message('error detected while processing {} at line "{}":\n{}'.format(_file_name(), line.rstrip(), str(e)))
 
     return None
+
+
+from NeoVintageous.plugin import PACKAGE_NAME
+from NeoVintageous.nv.vim import INSERT, INTERNAL_NORMAL, NORMAL, OPERATOR_PENDING, REPLACE, SELECT, UNKNOWN, VISUAL, VISUAL_BLOCK, VISUAL_LINE
+
+cfgU_settings = (f'{PACKAGE_NAME}.sublime-settings')
```

Comment:
Use the mains Preferences.sublime-settings for settings and the `get_setting` api consistent with other settings.

Reply:
but that pollutes the mains (NV is already ~third of my general settings file) and requires unwieldy long names with prefixes `neovintageous_search_occ_style`

Reply:
The `neovintageous` was mistake way back. The current prefix the old plugin prefix `vinstageous_`. I will probably be renaming the prefix later to something like `nv_`. See https://github.com/NeoVintageous/NeoVintageous/discussions/927.

Reply:
if you're going to change this anyway, all the more reason to use a proper plugin config file